spec/features/nav_menu_spec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+feature "nav menu" do
+  let(:community) { create(:community) }
+  let(:actor) { create(:user, community: community) }
+
+  around { |ex| with_user_home_subdomain(actor) { ex.run } }
+
+  before do
+    community.settings.main_nav_customizations = setting
+    community.save!
+    login_as(actor, scope: :user)
+  end
+
+  context "basic" do
+    let(:setting) { "" }
+    scenario "no customizations" do
```

Comment:
Will do.  As for the CI output, was the 9138_integrate_parser branch merged in?  If not, it would produce that error.   Also, I will switch the arguments to the equality test so that "expected" and "got" will be more intuitive.

Reply:
I think so. Please check?

Reply:
I checked, and the version of link_builder.rb in 9138_base does not include the changes that are in 9138_integrate_parser, which, I think, is why the CI failed.

Reply:
You're right. I think I fixed.

Reply:
Okay; I have pushed up the latest code.